src/Slider.jsx

```diff
@@ -8,11 +8,21 @@ import * as SliderConstants from './constants/SliderConstants';
 import linear from './algorithms/linear';
 
 function getClassName(props) {
-  const orientation = props.orientation === 'vertical'
+  let classNames = ['rheostat'];
+
+  classNames.push(props.orientation === 'vertical'
     ? 'rheostat-vertical'
-    : 'rheostat-horizontal';
+    : 'rheostat-horizontal');
+
+  if (props.disabled) {
+    classNames.push('rheostat-disabled');
+  }
 
-  return ['rheostat', orientation].concat(props.className.split(' ')).join(' ');
+  if (props.className) {
+    classNames = [...classNames, ...props.className.split(' ')];
```

Comment:
`classNames.push(...props.className.split(' '))`, but this should probably be using the `cx` package instead of manually splitting :-/